### ISSUE TITLE
Uninterpreted HTML in warning message of settings import

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#14908]
Using QA_4_8, I exported my settings, then tried to import them again. I got a warning message stating that some settings were invalid, and they included some literal `<i>` and `</i>` rather than rendering the text in italics.

![image](https://user-images.githubusercontent.com/1443710/43371217-65a897fe-935b-11e8-956f-5b3ba5a84760.png)

**Comment by bahl24** [comment]
@ibennetch I wasn't able to reproduce the issue, can you give detailed testing instructions?

**Comment by kartik1000** [comment]
@mauriciofauth can't we use the raw filter of twig in {{error}} part?

**Comment by ibennetch** [comment]
I missed the question from @bahl24 earlier, but I've seen it now. To reproduce (you may need to first have defined some custom setting in this settings tab):

Go to the Settings tab of the main page. The default tab is "Manage your settings". From "Export" pick "Save as file" (which should be "Save as JSON file", see #15115) - or create a custom JSON file from my example below. Save the JSON file to your local computer.

Now go to the "Import" area and "Browse" to the file you just downloaded, then press "Go" to begin the import. This message is shown afterwards.

My JSON file that reveals this problem:
```
{
    "NavigationLogoLink": "\/example.php",
    "Export\/method": "custom",
    "RowActionLinksWithoutUnique": true,
    "InitialSlidersState": "open",
    "Console\/Mode": "collapse",
    "Server\/hide_db": "",
    "Server\/only_db": ""
```

**Comment by williamdes** [comment]
Fixed by 2a4729c18ea8d04fc29db61b5c20d8d129b7708b
I missed the issue when creating the fix and the initial changelog entry
